### ISSUE TITLE
suggestion - compatablity mode

### ISSUE THREAD

**Issue by Bluers** [tr7zw/Exordium#85]
with the newer versions (1.1.0-1.19.4 and up) exdorium renders all vanilla hud separately (i think), and no longer renders all hud at the same fps.
While this is mostly a good thing, some people use custom hud/gui mods, which with the newer versions of the mod, there are no options to render them at a lower fps, losing some performance.
I suggest a "compatibility mode" which you can turn on and off, rendering other hud/gui at a custom fps. 

(im no mod dev so idk if this is possible but i would like for this to be considered)

**Comment by andreasdc** [comment]
It is configurable if I understood that well.

**Comment by tr7zw** [comment]
Exactly this is planned, a toggle that brings back the old logic.

**Comment by andreasdc** [comment]
> Exactly this is planned, a toggle that brings back the old logic.

Force update is not it?

**Comment by Felix14-v2** [comment]
> > Exactly this is planned, a toggle that brings back the old logic.
> 
> Force update is not it?

No, `Force update` forces the buffer to redrawing with selected FPS limit even if there were no changes in it. By default (`Force update` is off), Exordium renders a new frame only when something has been changed inside the buffer. For example, the previous version of exordium didn't handle enchantment glint, so enchanted items had no animation unless `Force update` was enabled. This is just an example, other mods can add things with the same effect, so this is mainly a compatibility feature.
